src/main/java/frc/robot/subsystems/Elevator/ElevatorIO.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.Elevator;
+
+import org.littletonrobotics.junction.AutoLog;
+
+/** Add your docs here. */
+public interface ElevatorIO {
+  @AutoLog
+  public static class ElevatorIOInputs {
+    public boolean switchPressed = false;
+    public double positionInches = 0.0;
+    public double percentOut = 0.0;
+    public double[] currentAmps = new double[] {};
+  }
+
+  public abstract void updateInputs(ElevatorIOInputs inputs);
```

Comment:
There's a few good reasons to avoid mutation methods and return a new object instead. First is that if you instead make this `getInputs` or something similar which returns a value, you can chain on that value vs needing to have a separate line for just updating the output variable. Second is that mutation can be a major pitfall when dealing with concurrent systems. Third is that if you want to unit test, mocking or stubbing out this result is much more complicated. Two of those three don't really apply in our codebase, but it's a useful habit to get into.

Honestly, the only good reason for mutation functions like this is if the object you're dealing with is particularly expensive to create. This doesn't look like that.

Strongly encourage you to replace this (and the analogous method in the other interfaces) to return a value instead.